### PR TITLE
Epoch UI: fix miscalculation when no reward is given

### PR DESCRIPTION
**Pull request by shazarre**
### Description

When no reward was given because of locked CELO revoked a block before epoch fetcher was miscalculating the reward. This PR aims to fix it.
 
### Tested

Covered by unit tests.

### Issues

 - Fixes https://github.com/celo-org/data-services/issues/574

### CODE REVIEW

File: apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex [hunk 1]
```diff
@@ -152,6 +152,13 @@ defmodule Indexer.Fetcher.CeloEpochData do
     Map.merge(block, %{voter_rewards: voter_rewards})
   end
 
+  def calculate_voter_rewards(after_rewards_votes, before_rewards_votes, _plus_revoked_minus_activated_votes)
+      when after_rewards_votes == before_rewards_votes,
```
Comment (carterqw2) [inline apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex:156]: Isn't this case handled by [this line](https://github.com/celo-org/blockscout/pull/814/files#diff-40d04a4c903621dde0aeebe53400212cc4327b1a244396f05ad7469e87aa1091R163)?

Comment (carterqw2) [inline apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex:156]: Isn't this case handled by [this line](https://github.com/celo-org/blockscout/pull/814/files#diff-40d04a4c903621dde0aeebe53400212cc4327b1a244396f05ad7469e87aa1091R163)?

Comment (shazarre) [reply apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex:156]: No, in this case particular case that I added a test for (when gold is revoked on the epoch block, but before the epoch rewards are distributed) it results in a negative result.

Comment (carterqw2) [reply apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex:156]: I think that must be a sign that we just incorrectly calculate `_plus_revoked_minus_activated_votes` and we should exclude the epoch block from it.

**Pull request merged**